doc/src/manual/strings.md

```diff
@@ -371,6 +401,34 @@ julia> string(greet, ", ", whom, ".\n")
 "Hello, world.\n"
 ```
 
+A situation which is important to be aware of is when invalid UTF-8 strings are concatenated.
+In that case the resulting string may contain different characters than those that constitute
+input strings and its number of characters may be lower than sum of numbers of characters
```

Comment:
"than those that constitute input strings and" --> "than the input strings, and"

Reply:
fixed